packages/amplify-nodejs-runtime-provider/src/utils/invokeOptions.ts

```diff
@@ -0,0 +1,7 @@
+export type InvokeOptions = {
+  packageFolder: string;
+  handler: string;
+  event: string;
+  context?: object;
+  environment?: any;
```

Comment:
it is a dictionary of string any aka js object

Reply:
updated type to `{[key: string]: string}`